gradle/publish.gradle

```diff
@@ -41,9 +41,11 @@ publishing {
 
 nexusPublishing {
     repositories {
+        // TODO: the following line causes the `The Project.getConvention() method has been deprecated.` warning.
         sonatype {
-            // //only for users registered in Sonatype after 24 Feb 2021
+            // only for users registered in Sonatype after 24 Feb 2021
             nexusUrl = uri("https://s01.oss.sonatype.org/service/local/")
+            snapshotRepositoryUrl = uri("https://s01.oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
Not sure if we need/want this. It's from the corresponding block in the readme: https://github.com/gradle-nexus/publish-plugin#publishing-to-maven-central-via-sonatype-ossrh

Reply:
We will if we want to publish snapshot releases (which I think we will want to, at least for the conformance tests). I'm doing the same thing in my work to decouple the builds.

Reply:
Undid these changes to avoid conflicts with #446.